includes/calendars/views/default-calendar-grid.php

```diff
@@ -164,6 +166,20 @@ public function styles()
 		];
 	}
 
+	/**
+	 * Format the timestamp with timeZone to display exact month as per the diffrent timezone.
+	 *
+	 * @since  3.4.1
+	 */
+	public function format_timestamp($dateformat = 'F', $timestamp)
```

Comment:
Why `$dateformat`? It could be time format as well. Better to make it generic like `$format`

Reply:
done